### ISSUE TITLE
Publish package with non-MIT license

### ISSUE THREAD

**Issue by benlau** [Cutehacks/qpm#7]
Can I publish a package not using MIT license? I have tried to change license to Apache/APL but it will just complains:

```
$ qpm publish
Running check
QPM: 2015/10/06 14:44:18 ERROR: json: cannot unmarshal string into Go value of type messages.LicenseType
QPM: 2015/10/06 14:44:18 json: cannot unmarshal string into Go value of type messages.LicenseType
```

**Comment by mariusbu** [comment]
Hi, we only support MIT license at the moment, but support for more licenses is on the roadmap (see https://trello.com/c/k2w1f5tm). It's a high priority item for us, but feel free to vote or add suggestions to the backlog.

**Comment by jrbarron** [comment]
@benlau I released version 0.10.0 tonight so I think this should be working now.

**Comment by benlau** [comment]
Thx!
